packages/engine/src/experiment/controller/controller.rs

```diff
@@ -393,7 +393,7 @@ impl<P: OutputPersistenceCreatorRepr> ExperimentController<P> {
         exp_config: Arc<ExperimentConfig>,
         exp_base_config: Arc<ExperimentConfig>,
         env: Environment,
-        shared_store: Arc<SharedStore>,
+        shared_store: Arc<SharedDatasets>,
```

Comment:
You also haven't renamed variables and such where this is used which is confusing. I'm thinking we might just want to revert this commit